src/games/GameDetails.js

```diff
@@ -1,53 +1,52 @@
 import { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
 import { useParams } from "react-router-dom";
-import PugApi from "../api/api";
 import LoadingSpinner from "../common/LoadingSpinner";
+import { fetchGame, fetchGames } from "../store/games/gamesSlice";
 
 function GameDetails() {
-    
-  const { gameId }= useParams();
-  console.debug("GameDetail", "gameId=", gameId);
-
-  const [game, setGame] = useState(null)
-  const [details, setDetails] = useState(null)
-  const [comments, setComments] = useState(null)
-  const [players, setPlayers] = useState(null)
-
-  useEffect(()=>{
-    async function getGame(){
-      const game = await PugApi.getGame(gameId) 
-      const {details, comments, players } = game
-      setGame(game)
-      setDetails(details)
-      setComments(comments)
-      setPlayers(players)
-    }
+  const { gameId } = useParams();
 
-    getGame()
-  }, [gameId])
+  const dispatch = useDispatch();
+  const gameStatus = useSelector((state) => state.games.status);
+  const games = useSelector((state) => state.games.entities);
```

Comment:
you can use this selector to get the exact game you need

Reply:
updated